tests/test_api.py

```diff
@@ -1812,10 +1823,10 @@ def test_reactor_config_secret(self):
         ("plat", 'rel', None, True, False, 1, False),
         ("plat", 'rel', None, True, False, None, True),
         # orchestrator build
-        (None, None, 'platforms', False, True, 1, True),
+        (None, None, 'platforms', False, True, 1, False),
```

Comment:
arrangement_version is optional param for orchestrator, at least according docstring

- arrangement_version param may be used to select which                       
  orchestrator_inner:n.json template to use